test/functional/rpc_net.py

```diff
@@ -386,5 +391,70 @@ def test_getaddrmaninfo(self):
             assert_equal(res[net]["tried"], 0)
             assert_equal(res[net]["total"], 0)
 
+        self.log.debug(
+            "Test that verbose keys are not in the default getaddrmaninfo")
+        assert "new_table" not in res
+        assert "tried_table" not in res
+
+        self.log.info("Test getaddrmaninfo verbose = True")
+        res = node.getaddrmaninfo(verbose=True)
+
+        # we expect one addrman new table entry added in a previous test
+        assert_equal(len(res["new_table"]), 1)
+        assert_equal(len(res["new_table"]), res["all_networks"]["new"])
+        assert_equal(res["new_table"][0]["address"], "2.0.0.0:8333")
+        assert_equal(res["new_table"][0]["services"], 9)
+        assert_equal(res["new_table"][0]["source"], "2.0.0.0")
+        assert 0 <= res["new_table"][0]["bucket"] < ADDRMAN_NEW_BUCKET_COUNT
+        assert 0 <= res["new_table"][0]["position"] < ADDRMAN_BUCKET_SIZE
+
+        # we expect one addrman tried table entry added in a previous test
+        assert_equal(len(res["tried_table"]), 1)
+        assert_equal(len(res["tried_table"]), res["all_networks"]["tried"])
+        assert_equal(res["tried_table"][0]["address"], "1.2.3.4:8333")
+        assert_equal(res["tried_table"][0]["services"], 9)
+        assert_equal(res["tried_table"][0]["source"], "1.2.3.4")
+        assert 0 <= res["tried_table"][0]["bucket"] < ADDRMAN_TRIED_BUCKET_COUNT
+        assert 0 <= res["tried_table"][0]["position"] < ADDRMAN_BUCKET_SIZE
+
+        self.log.debug("Add one new address to each addrman table")
```

Comment:
In da44fd3e499ca53242332ef9cb2907ec31fce1e3: I think we could addresses from other networks (e.g tor) to test it as well.

Reply:
will do if I re-touch the tests